tests/js/tests/prolog-and-literals.test.ts

```
@@ -782,6 +794,48 @@ describe("Prolog + Literals", () => {
+                        target: imageExprUrl
+                    }));
+
+                    // Verify the link was created with the expression URL
+                    //@ts-ignore
+                    let links = await perspective!.get(new LinkQuery({source: root, predicate: "recipe://image"}));
+                    expect(links.length).to.equal(1);
+                    expect(links[0].data.target).to.equal(imageExprUrl);
+
+                    // Retrieve the recipe and verify the image was resolved and transformed
+                    const results = await Recipe.findAll(perspective!, { where: { name: "Test with image" } });
+                    await recipe2.get();
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)